src/v2/components/Validators/Table/index.jsx

```diff
@@ -70,32 +68,33 @@ const ValidatorsTable = ({vertical}: {vertical: boolean}) => {
     <div className={classes.root}>
       <div className={classes.header}>
         <Typography>Validators</Typography>
-        <Typography variant="h5">{nodes.length}</Typography>
-        <Link to="validators/all" className={classes.link}>
-          See all &gt;
-        </Link>
+        <Typography variant="h5">{voting.length}</Typography>
+        {!separate && (
+          <Link to="validators/all" className={classes.link}>
```

Comment:
I think these links should be absolute if possible because there is cross-linking from detail to table to overview etc.

Reply:
fixed